pkg/metallb/mlbtypes/bpgpeer.go

```diff
@@ -0,0 +1,94 @@
+package mlbtypes
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type MatchExpression struct {
+	Key      string `json:"key"`
+	Operator string `json:"operator"`
+	// +kubebuilder:validation:MinItems:=1
+	Values []string `json:"values"`
+}
+
+type NodeSelector struct {
+	// +optional
+	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+
+	// +optional
+	MatchExpressions []MatchExpression `json:"matchExpressions,omitempty"`
+}
+
+// BGPPeerSpec defines the desired state of Peer.
+type BGPPeerSpec struct {
+	// AS number to use for the local end of the session.
```

Comment:
ASN number?

Reply:
file was simple copy/paste from upstream metallb project.  I'll not change it because we are going to sync this file periodically